src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -204,7 +222,7 @@ object BulkCopyUtils extends Logging {
         val colMetaData = {
             if(checkSchema) {
                 checkExTableType(conn, options)
-                matchSchemas(df, rs, options.url, isCaseSensitive)
+                matchSchemas(conn, options, df, rs, options.url, isCaseSensitive)
```

Comment:
Why full option parameter passing is required? Only dbtable name is required by matchScema

Reply:
Changed. Now using dbtable instead of option parameter.